docs/products/clickhouse/howto/data-service-integration.md

```diff
@@ -1,134 +1,240 @@
+
+To create an integration with an external data source in the **Data service integrations**
+wizard:
+
+1. Select **Service types** of an external data source to integrate with. See
+   [supported data source types](/docs/products/clickhouse/concepts/data-integration-overview#supported-data-source-types)
+   for supported service types.
+1. Select an external **Data service** of the chosen type from the list of services available
+   for integration, and click **Continue**.
+1. Select an integration method, either **Managed databases**  or **Managed credentials**,
+   and click **Continue**.
+1. Integrate with managed databases or with managed credentials:
+
+<Tabs groupId="group2">
+<TabItem value="1" label="Integrate with managed databases" default>
+The managed databases integration uses databases engines and, when enabled,
+automatically creates databases in your Aiven for ClickHouse, where you can ingest your
+external data.
+
+1. Select **Managed databases**, and click **Continue**.
+1. Click **Add databases** to have custom databases created along with the integration, or
+   click **Enable without databases** to integrate with no custom databases created.
+
+   :::tip
+   If you choose to have databases created automatically, you can query them using a
+   statement similar to the following:
+
+   ```sql
+   SELECT *
+   FROM EXTERNAL_POSTGRESQL_RESOURCE_NAME.POSTGRESQL_TABLE_NAME
+   ```
+
+   :::
+
+</TabItem>
+<TabItem value="2" label="Integrate with managed credentials">
+The managed credentials integration supports storing connection parameters in Aiven
+and allows you to create tables for your external data. The stored connection parameters
+are automatically seeded in your external data queries.
+
+1. Select **Managed credentials**, and click **Continue** > **Enable**.
+1. Create tables using
+   [table engines](/docs/products/clickhouse/reference/supported-table-engines), for
+   example the PostgreSQL engine:
+
+   ```sql
+   CREATE TABLE default.POSTGRESQL_TABLE_NAME
+   (
+      `float_nullable` Nullable(Float32),
+      `str` String,
+      `int_id` Int32
+   )
+   ENGINE = PostgreSQL(postgres_creds);
+   ```
+
+   :::tip
+   For details on how to use different table engines for integrations with external
+   systems, see the
+   [upstream ClickHouse documentation](https://clickhouse.com/docs/en/engines/table-engines/integrations).
+   :::
+
+Depending on the type of data source you are integrated with, you can access your credentials
+storage by passing your data source name in the following query:
+
+```sql
+# PostgreSQL data source
+
+SELECT *
+FROM postgresql(
+   `service_POSTGRESQL_SOURCE_NAME`,
+   database='defaultdb',
+   table='tables',
+   schema='information_schema'
+)
+```
+
+```sql
+# MySQL data source
+
+SELECT *
+FROM mysql(
+   `service_MYSQL_SOURCE_NAME`,
+   database='defaultdb',
+   table='schemata'
```

Comment:
Tested out the query for mysql with @joelynch, figure out that we had to modify the query.
This one works out of the box
```suggestion
   database='mysql',
   table='slow_log'
```